<?xml version="1.0" encoding="utf8"?>

<launch>
  <node pkg="robot_control" type="motor_calibrate01.py" name="motor_calibrate01" output="screen">
    <param name="param_path" value="$(find robot_control)/params/params.yaml"/>
  </node>
</launch>
